_includes/image.html

```diff
@@ -0,0 +1,44 @@
+{% comment %}
+<!--
+  Image Include
+
+  This partial is used as a "prequel" include that is used with the responsive
+  image template.  This partial's main purpose is to dynamically create the
+  full path for the image to then pass that path as well as the other
+  parameters to the responsive image plugin and template.
+
+  @param {string} "file" - Required. filename of the image.
+  @param {string} "alt" - Required. Alt attribute for image tag.
+  @param {boolean} "img_link" - Optional, default is false. If img should have
+    the image overlay functionality that the theme provides.
+  @param {string} "fig_caption" - Optional. Caption to be used with figure tag.
+  @param {string} "media_rendition" - Optional.  The title for the array of
+    media queries to include for the "sizes" attribute on image tag that is
+    listed in the global _config file. If it's not included then defaults to
+    "article".
+
+-->
+{% endcomment %}
+
+{% comment %}
+<!-- determine the full path of the image -->
+{% endcomment %}
+{% assign post_slug =  page.path | replace:'.html','' | replace: '.md','' | replace: '_posts/', '' %}
+{% assign img_basepath = site.responsive_image.base_path | append: "/" | append: post_slug %}
+{% assign img_path = img_basepath | append: "/" | append: "full-traffic-graph.png" %}
```

Comment:
You are absolutely correct, that should not be in there...hard-coding mistake developing the solution.

That is fixed in c1e9aee.